### ISSUE TITLE
prime-gaming email locator not found

### ISSUE THREAD

**Issue by saschabrockel** [vogler/free-games-claimer#401]
![image](https://github.com/user-attachments/assets/4fddd2b1-076a-4d1b-8211-79ebf43dd2cd)

```
Not signed in anymore.
--- Exception:
page.fill: Timeout 180000ms exceeded.
Call log:
  - waiting for locator('[name=email]')

    at /fgc/prime-gaming.js:55:18 {
  name: 'TimeoutError'
}
```

Is this new? I have no idea.

**Comment by zang74** [comment]
I was logged out as well. And annoyingly "Sign in" doesn't seem to do anything but reload the page.

The trick I found was hijacking No-VNC with a new tab, using it to go to "http://amazon.com/your-account". Even if shows you as  logged in, it will ask you for your password as confirmation. Once I hit enter and was able to see my account info, I went over to http://gaming.amazon.com in that same browser tab and it showed as logged in. I then just re-ran the scripts and all was good.
